circle.yml

```diff
@@ -14,6 +14,6 @@ dependencies:
 
 test:
   override:
-    - istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly test/* --reporter --recursive -- -R spec
+    - istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly test/* test/**/**/* --reporter --recursive -- -R spec
```

Comment:
Change this to `istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly test/* test/**/**/* --reporter spec --recursive`